### PR TITLE
Update README.adoc

### PR DESCRIPTION
**Pull request by nateziemann**
Updated introduction.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -8,18 +8,21 @@ permalink: /guides/app-logging/
...
+Pod processes running in Kubernetes frequently produce logs. To effectively manage this log data and ensure no loss of log data occurs when a pod terminates, a log aggregation tool should be deployed on the Kubernetes cluster. Log aggregation tools help users persist, search, and visualize the log data that is gathered from the pods across the cluster. Log aggregation tools in the market today include:  EFK, LogDNA, Splunk, Datadog, IBM Operations Analytics, etc.  When considering log aggregation tools, enterprises will make choices that are inclusive of their journey to cloud, both new cloud native applications running in Kubernetes and their existing traditional IT choices. 
+
```
Comment (donbourne) [inline README.adoc:18]: last sentence in paragraph above is a bit hard to parse.

**Review by donbourne — APPROVED** [review]

**Review by fwji — APPROVED** [review]
